boards/arm/lpcxpresso55s36/lpcxpresso55s36.dts

```diff
@@ -168,3 +170,15 @@ zephyr_udc0: &usbfs {
 	pinctrl-0 = <&pinmux_usbfs>;
 	pinctrl-names = "default";
 };
+
+&dma0 {
+	dma-channels = <52>;
```

Comment:
dma-channels should be set at SOC level (see activity in #59767, this is a bug being currently fixed, don't add to it)

Reply:
I have moved these to the SOC level